docs/requirements.txt

```diff
@@ -1,4 +1,4 @@
-mkdocs==1.3.1
+mkdocs==1.2.4
```

Comment:
@cmsirbu can you take a look.

Reply:
@itdependsnetworks we are just testing something with @cmsirbu